src/scripts/main.js

```diff
@@ -1 +1,24 @@
 'use strict';
+
+const nav = document.querySelector('.list-item');
+const imgLarge = document.querySelector('.gallery__large-img');
+const a = document.querySelector('a');
+const img = document.querySelector('.gallery__thumb');
+
+a.addEventListener('click', () => {
+  event.preventDefault();
+});
+
+img.addEventListener('click', () => {
+  event.preventDefault();
+});
+
+nav.addEventListener('click', () => {
+  const link = event.target.closest('.list-item__link', '.gallery__img');
```

Comment:
The `closest` method should only take one argument, which is a selector string. The current usage with two arguments is incorrect. You might want to check if the target is either a '.list-item__link' or '.gallery__img'.